test/ebpf/src/endpoint_prog.c

```diff
@@ -65,7 +80,7 @@ _policy_eval(bpf_sock_addr_t *ctx, uint32_t compartment_id, policy_map_key_t key
     if (verdict != NULL)
     {
         char msg[128];
-        bpf_snprintf(msg, sizeof(msg), "Policy Eval: L4 policy ID %d Allowed.", *verdict, sizeof(*verdict));
+        bpf_snprintf(msg, sizeof(msg), "Policy Eval: L4 policy ID %lu Allowed.", (unsigned long *) verdict, sizeof(verdict));
         print_trace(ctx, msg);
```

Comment:
Can't you simply do:
```suggestion
        bpf_printk("Policy Eval: L4 policy ID %lu Allowed.", *(unsigned long *) verdict);
```